plugins/metrics/parser/src/metricsparser.cpp

```diff
@@ -51,6 +51,40 @@ std::vector<std::string> MetricsParser::getDependentParsers() const
   return std::vector<std::string>{};
 }
 
+bool MetricsParser::preparse()
+{
+  if(!_fileIdCache.empty())
+  {
+    try
+    {
+      (util::OdbTransaction(_ctx.db))([this] {
+        for(const model::File& file
+          : _ctx.db->query<model::File>(
+          odb::query<model::File>::id.in_range(_fileIdCache.begin(), _fileIdCache.end())))
+        {
+          auto it = _ctx.fileStatus.find(file.path);
+          if(it != _ctx.fileStatus.end() &&
```

Comment:
space

Reply:
Fixed in 64cb407581662e78675baf916a95dba2b765b3e4.